tests/cellfinder_core/test_unit/test_cellfinder.py

```diff
@@ -116,53 +164,14 @@ def test_add_signal_and_background_files(
     message_pattern : str
         Expected pattern in the log
     """
-    # instantiate our custom logger
+
+    # instantiate custom logger
     _ = setup_logger()
 
     # read json as Cellfinder config
-    config = read_cellfinder_config(input_configs_dir / input_config)
-
-    # monkeypatch cellfinder config:
-    # set install_path to pytest temporary directory
-    config.install_path = tmp_path / config.install_path
-
-    # check lists of signal and background files are not defined
-    assert not (config.list_signal_files and config.list_background_files)
-
-    # build fullpaths to input data directories
-    config.signal_dir_path = str(
-        Path(config.install_path)
-        / config.data_dir_relative
-        / config.signal_subdir
-    )
-    config.background_dir_path = str(
-        Path(config.install_path)
-        / config.data_dir_relative
-        / config.background_subdir
-    )
-
-    # monkeypatch cellfinder config:
-    # if config is "local" or "signal/background missing":
-    # ensure signal and background data from GIN are downloaded locally
-    if input_config in [
-        "input_data_locally.json",
-        "input_data_missing_signal.json",
-        "input_data_missing_background.json",
-    ]:
-        # fetch data from GIN and download locally
-        pooch.retrieve(
-            url=cellfinder_GIN_data["url"],
-            known_hash=cellfinder_GIN_data["hash"],
-            path=config.install_path,  # path to download zip to
-            progressbar=True,
-            processor=pooch.Unzip(
-                extract_dir=config.data_dir_relative
-                # path to unzipped dir, *relative*  to 'path'
-            ),
-        )
-
-    # add signal and background files lists to config
-    add_signal_and_background_files(config)
+    # ---> change so that the fixture is the config object!
+    # config = read_cellfinder_config(input_configs_dir / input_config)
```

Comment:
Just double-checking - how are we making sure we don't forget this desired refactor?
Issue, or do you have notes somewhere that this is a TODO for your next PR, or ...?